contracts/test/ERC20.t.sol

```diff
@@ -124,6 +139,30 @@ contract ERC20BaseTest is TestUtils {
         }
     }
 
+    function testFuzz_transferFrom_infiniteApproval(address recipient_, uint256 amount_) public {
+        amount_                  = constrictToRange(amount_,   0, type(uint256).max);
+        uint256 infiniteApproval = type(uint256).max;
+        if (amount_ > infiniteApproval) return;  // Owner must approve for more than amount.
```

Comment:
When would this condition trigger?

Reply:
Never, this was accidently left in when fixing the test

Reply:
Lets remove this line